packages/core/src/createConfig.ts

```diff
@@ -40,9 +40,14 @@ type ConfigParams = {
 
 	/**
 	 * The data provider to use
-	 * @default MempoolSpaceProvider
+	 * @default MaestroProvider
 	 */
 	provider?: AbstractProvider;
+
+	/**
+	 * API key for MaestroProvider (if used)
+	 */
+	maestroApiKey?: string;
```

Comment:
This is a provider specific configuration and shouldn't be added to the core type